app/models/talk.rb

```diff
@@ -206,7 +206,7 @@ def thumbnail(size = :thumbnail_lg)
       end
     end
 
-    if event && (asset = Rails.application.assets.load_path.find(event.poster_image_path)) && !youtube?
+    if !youtube? && event && (asset = Rails.application.assets.load_path.find(event.poster_image_path))
```

Comment:
Is this for delaying the event lookup and fetch?

Reply:
Yes I noticed that we very frequently query the event but most of the time we don’t need to. If we are in a YouTube talk thé we can move directly to the next part without making this extra query 

this method is a bit complex so let me know if I missed something

Reply:
That looks good, was just wondering because it looked unrelated at first. But I think this is good 👍🏼